src/test/java/com/googlecode/jsonrpc4j/server/JsonRpcBasicServerTest.java

```diff
@@ -0,0 +1,134 @@
+package com.googlecode.jsonrpc4j.server;
```

Comment:
seems you removed lot of tests in the previous class :) don't like that part, why?